frontend/pages/404.tsx

```diff
@@ -0,0 +1,44 @@
+import { Center, Box, Text, Button, Container, Flex } from '@chakra-ui/react';
+import { DEFAULT_STORE_CODE } from '@config/env';
+import { DefaultLayout, DefaultLayoutProps } from '@layouts/default';
+import { getGlobalSettings } from '@models/global-settings';
+import { getStoreByCode, getStoreList } from '@models/store';
+
+export default function Custom404({
+   currentStore,
+   stores,
+   globalSettings,
+}: DefaultLayoutProps) {
+   return (
+      <DefaultLayout {...{ currentStore, stores, globalSettings }}>
```

Comment:
What you have works, but this might be a little more readable.

```suggestion
export default function Custom404(layoutProps: DefaultLayoutProps) {
   return (
      <DefaultLayout {...layoutProps}>
```